.snyk

```diff
@@ -6,4 +6,8 @@ ignore:
     - '*':
         reason: Fix not available
         expires: 2020-12-15T00:00:00.000Z
+  SNYK-JAVA-ORGGLASSFISHJERSEYMEDIA-595972:
+    - '*':
+        reason: Not used directly by us; Fix not available
+        expires: 2020-09-07
```

Comment:
How do we know if a fix will be available around this time?  Wouldn't say 3 months be a better length of time for this to expire?

Reply:
Wishful thinking on my part / better to have to do it again in a month than not realise it's been fixed and have it unpatched for two months (assuming the fix becomes available).

Reply:
Cool... I get that... just felt that it breaking again in a month was a bit sad if we could avoid it :-) :+1: